train.py

```diff
@@ -35,11 +39,39 @@
 from utils.torch_utils import ModelEMA, select_device, intersect_dicts, torch_distributed_zero_first, is_parallel
 from utils.wandb_logging.wandb_utils import WandbLogger, check_wandb_resume
 
+# TODO: Rip out all Weights and Biases References
```

Comment:
Maybe replace the TODOs with github issues

Reply:
Done